tslint.json

```diff
@@ -35,6 +35,107 @@
 				"messageIndex": 1
 			}
 		],
+		"import-patterns": [
+			true,
```

Comment:
SQL CARBON EDIT?

Reply:
I added a comment above to describe the differences here between vscode and us